Notes/Strasnote.Notes.Data.Entities/UserEntity.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Strasnote
+// Licensed under https://strasnote.com/licence
+
+using System.Collections.Generic;
+
+namespace Strasnote.Notes.Data.Entities
+{
+	/// <summary>
+	/// User entity
+	/// </summary>
+	public sealed class UserEntity : Auth.Data.Entities.UserEntity, IEntity
+	{
+		/// <summary>
+		/// User ID (alias for <see cref="Microsoft.AspNetCore.Identity.IdentityUser{TKey}.Id"/>)
+		/// </summary>
+		public long UserId
+		{
+			get => Id;
+			set => Id = value;
+		}
+
+		/// <summary>
+		/// The User's given name (or Christian / first name)
+		/// </summary>
+		public string UserGivenName { get; init; } = string.Empty;
```

Comment:
Do we need these details?

Reply:
I like e.g. WordPress's 'Hi Ben!' message - happy not to have it though!

Reply:
Yeah we can do that, but they should be nullable. so when signing up (which will have to be done whether it's self hosted or not) they would only be asked for an email and password. then they can go into a profile page later on (or during the wizard setup) and enter an optional name